app/lib/irs1040_pdf.rb

```diff
@@ -1,15 +1,15 @@
-class AdvCtcIrs1040Pdf
```

Comment:
I sort of wish we'd followed the pattern like we did for the schemas where we kept last years stuff around, namespaced them to the year, and created a new one... just in case we do file backtaxes this season, but I feel like that likely won't happen so this is cool.